src/components/block/FooterCTACard/FooterCTACard.module.css

```diff
@@ -0,0 +1,24 @@
+.body {
+  font: var(--font-body-medium);
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 15px;
```

Comment:
Would it be better to use the shorthand margin property? For example, `margin: [top] [left-and-right] [bottom];` or in this case, `margin: 15px auto 0px;`.